src/components/Pagination/Pager.tsx

```diff
@@ -238,7 +239,7 @@ export const Pager: FC<PagerProps> = React.forwardRef(
                             />
                         </li>
                     )}
-                {pageCount > 1 && (
+                {pageCount > 1 && showLast && (
```

Comment:
Is the purpose of this that jumping this many pages does not work well with the specific DB being used? Or is it more about the last page being undefined in that DB? If it's the latter, I recommend tracking this in the array of pages and excluding the last. Say we get 100 pages - if showLast is false, this button would be page 99.